hibernate-core/src/main/java/org/hibernate/loader/entity/CacheEntityLoaderHelper.java

```diff
@@ -109,7 +109,7 @@ public PersistenceContextEntry loadFromSessionCache(
 	 * of entity hierarchies.
 	 * <p/>
 	 * If checkDeleted was set to true, then if the entity is found in the
-	 * session-level cache, it's current status within the session cache
+	 * session-level cache, its current status within the session cache
```

Comment:
this one was actually correct ;)  (in this context the possessive form is required, i.e its, not the shorthand for 'it is')